csfieldguide/templates/base.html

```diff
@@ -137,7 +137,7 @@ <h1>
                     <img class="img-fluid py-5" src="{% static 'img/logos/uc-computer-science-education-logo.png' %}">
                   </a>
                 </div>
-                <div class="col-12 col-sm-6 col-md-4 text-center">
+                <div class="col-12 col-sm-6 col-md-4 logo-mobile-center">
```

Comment:
This seems to do nothing new. Can you please explain it's purpose?

Reply:
I couldn't find a bootstrap class that would apply the `text-center` class for viewports size SM or _smaller_. I could only find `text-sm-center` which will apply for viewports of size SM or _larger_. Do you know how to achieve this with built in bootstrap classes?

Reply:
Bootstrap has appeared (can't find logic for decision) to only do viewports going up. Either an earlier version or another library I used (Bootstrap 2 or MaterializeCSS) had both directions but this can be confusing.

Regardless, they way to do it is `text-center text-md-left`. Theoretically it would mean we don't need to include the new rule and save a few bytes in the CSS file, but up to you if you want to change it.

Reply:
Change made! :)